aytests/sles12.xml

```diff
@@ -826,6 +827,20 @@ ls -l /mnt/$BASE
     </user>
     <user>
       <encrypted config:type="boolean">false</encrypted>
+      <fullname>suse</fullname>
+      <gid>100</gid>
+      <home>/home/suse</home>
+      <shell>/bin/bash</shell>
+      <uid>1001</uid>
+      <user_password>nots3cr3t</user_password>
+      <username>suse</username>
+      <authorized_keys config:type="list">
```

Comment:
That's a new feature :-) (implements [here](https://github.com/yast/yast-users/pull/113))